src/pages/resultsView/plots/styles.scss

```diff
@@ -34,10 +34,32 @@
     .axisBlock {
       background:#eee;
       border-radius:4px;
+      padding-bottom: 0;
     }
+  } 
+}
+
+.tab-title {
```

Comment:
just noticed something else.  these new style classes will actually be global to project and could affect things inadvertantly since they have generic names (e.g. label-text).   The sylesheets are not modular in this way even though it seems they should be.  So you should copy these classes inside the .plotsTab class above it.  then they will ONLY be applied to elements inside .plotsTab div